modules/core/src/v2/coins/eos.ts

```diff
@@ -91,6 +93,29 @@ interface ExplainTransactionOptions {
   headers: EosTransactionHeaders;
 }
 
+interface SendableTransaction {
+  compression: string;
+  packed_trx: string;
+  signatures: string[];
+}
+interface RecoveryTransaction extends SendableTransaction {
```

Comment:
Why extend sendabletransaction but also have a sendable transaction inside? Maybe RecoveryTransaction should be a basecoin class, extended here as EosRecoveryTransaction